cypress/e2e/tests/migration/applicationinventory/assessment/assess_review.test.ts

```diff
@@ -64,6 +64,8 @@ describe(["@tier1"], "Application assessment and review tests", () => {
         application.verifyStatus("assessment", "Completed");
         application.validateAssessmentField("Low");
 
```

Comment:
Could you add Polarion TC before the test starts ?
Polarion MTA-517

Reply:
done!